src/handlers/tokenSearch.ts

```diff
@@ -56,4 +57,24 @@ const tokenSearch = async (searchParams: {
   }
 };
 
-export default tokenSearch;
+export const walletFilter = async (params: {
+  addresses: EthereumAddress[];
+  departureChainId: number;
+  destinationChainId: number;
```

Comment:
any reason to change the formatting of these names?

https://github.com/rainbow-me/rainbow/pull/4192/files#diff-fc39c286bdc8768f081731a684b5ed51a1c3c6c31125422376fbf98c12090fabR22

`fromChainId` and `toChainId` lgtm

Reply:
Because `toChainId` and `fromChainId` are flipped depending on whether you are filtering the user's wallet assets vs requesting filtered tokens via search requests, I found it sort of unintuitive and confusing, so I named these something different to make it clear.

I'm fine with switching these to `fromChainId` and `toChainId`, though.

Reply:
perfect, sounds good then!

Reply:
okay this has been changed to `toChainId` and `fromChainId`